server/webapp/resources/project.py

```diff
@@ -951,3 +951,27 @@ def get(self, project_id):
             "categories": program_categories
         }
         return payload
+
+
+# It's a pship but it's used somewhat interchangeably with populations
+
+pship_fields = {
+    "type": fields.String,
+    "populations": Json
+}
+
+
+class Partnerships(Resource):
+    @swagger.operation(
+        summary="List partnerships for project"
+    )
+    @report_exception
+    @marshal_with(pship_fields, envelope="populations")
+    @login_required
+    def get(self, project_id):
+        project = load_project(project_id, raise_exception=True)
```

Comment:
Let's perhaps keep to the convention that xxx_entry is record and xxx_instance is hydrated thing?